chain/beacon/drand/drand.go

```diff
@@ -100,7 +101,9 @@ func NewDrandBeacon(genesisTs, interval uint64, ps *pubsub.PubSub, config dtypes
 		}
 		hc.(DrandHTTPClient).SetUserAgent("drand-client-lotus/" + build.NodeBuildVersion)
 		clients = append(clients, hc)
-
+	}
+	if len(clients) == 0 {
```

Comment:
```suggestion
	if ps != nil && len(clients) == 0 {
```

Could also do this, leave the older beacons alone if they at least have a pubsub client to work with; but maybe we should just be more strict and even go as far as only adding pubsub for clients where we don't specify `Servers`, i.e. use `Servers` to tells us whether we expect to be able to fetch entries for this beacon or not.

I'm just not sure what pubsub is giving us here, is it used as a fallback client?